test/integration/029_docs_generate_tests/test_docs_generate.py

```diff
@@ -1413,7 +1404,6 @@ def expected_bigquery_complex_manifest(self):
                     },
                     'sources': [],
                     'depends_on': {'macros': [], 'nodes': ['seed.test.seed']},
-                    'empty': False,
```

Comment:
We never wanted this to be part of the schema, it was purely derived information.